README.md

```diff
@@ -170,6 +170,7 @@ Asynchronous concurrent distributed networking framework for PHP
 
 * [amphp/mysql](https://github.com/amphp/mysql) - An async mysql client built on [amp](#amphp)
 * [amphp/pgsql](https://github.com/amphp/pgsql) - A non-blocking PostgreSQL library built on [amp](#amphp)
+* [amphp/redis](https://github.com/amphp/redis) - An async redis client built on [amp](#amphp)
```

Comment:
I could only find one instance of a duplicate after checking just now, which I've addressed by consolidating the duplicates into a separate section in 2c3bf5c. Thanks for pointing this out.

As I said, I think listing all Redis clients in either Databases or Caching is reasonable, but not both.